packages/react-components/react-breadcrumb/src/components/Breadcrumb/useBreadcrumbContextValue.ts

```diff
@@ -0,0 +1,5 @@
+import type { BreadcrumbContextValue, BreadcrumbState } from './Breadcrumb.types';
+
+export function useBreadcrumbContextValue(props: BreadcrumbState): BreadcrumbContextValue {
+  return props;
+}
```

Comment:
This should be memoized to prevent unnecessary:

```suggestion
export function useBreadcrumbContextValue(state: BreadcrumbState): BreadcrumbContextValues {
  const { size, dividerType } = state

  const breadcrumb = React.useMemo(() => ({ size, dividerType }, [size, dividerType])
  
  return { breadcrumb }
}
```

For reference: `packages/react-components/react-accordion/src/components/AccordionItem/useAccordionItemContextValues.ts`